guides/release/testing/testing-components.md

```diff
@@ -74,7 +90,7 @@ module('Integration | Component | pretty-color', function(hooks) {
     // set the outer context to red
     this.set('colorValue', 'red');
```

Comment:
Should this still be `set`?

Reply:
yes I tested all the examples in this whole thing... took me way longer than I had allocated to do this 🙈 🫠 

Essentially if we don't have this.set then we need to get people to initialise a class with tracked on one of the fields in every one of these tests. Maybe we might want to teach people cell (probably not) but pedagogically it's better to keep this simple and unchanged for now. We can solve this question more substantially later 👍

Reply:
I just worry we won't revisit it since that RFC is considered part of the overall template tag RFC and we'll say that 'template tag in the guides is done'